packages/core/lib/Configuration.ts

```diff
@@ -157,6 +157,15 @@ export class Configuration {
   configureStorageOptions<T>(yargs: Yargs.Argv<T>) {
     return yargs.options({
       // directories
+      "syntest-directory": {
```

Comment:
What is saved in this directory at the moment, If we keep both the final test cases and the statistics separate?

Reply:
Nothing, it just specifies the main folder.

Reply:
However, if the tests and the statistics are stored in their own separate path, what is the use case for this  `syntest-directory`?

Reply:
It's mostly meant for plugin to be able to use the main folders.
I think its still valuable to be able to pick other locations for the statistics and test cases.